src/expandableCalendar/index.tsx

```diff
@@ -255,29 +259,35 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
     }
   }, []);
 
-  useEffect(() => {
-    // date was changed from AgendaList, arrows or scroll
-    scrollToDate(date);
-  }, [date]);
-
   const handleScreenReaderStatus = (screenReaderEnabled: any) => {
     setScreenReaderEnabled(screenReaderEnabled);
   };
 
   /** Scroll */
+  const autoScroll = useRef(false);
```

Comment:
Move to the top of the function, near the other refs

Reply:
The refs are placed by related section, not only on top. I thought it will be more readable this way...

Reply:
I see. 
I mention it before, but you should consider moving each section's code to a hook. 
You'll see how it makes the code even more readable 
You can have useScrollHandlers, useGesturesHandler, etc.. 
Anyway, not related to this PR